lib/Interpreter/CppInterOp.cpp

```diff
@@ -3277,7 +3278,36 @@ namespace Cpp {
     int m_DupFD = -1;
 
   public:
+#if defined(_WIN32)
```

Comment:
We can use #define _dup dup and the same for others. I’ve seen that’s what folks do for such type of code.

Reply:
Done